website/docs/rbs-support.md

```diff
@@ -63,49 +63,12 @@ an alternative annotation syntax for Ruby. The headline features of RBS:
 
 - It's the type annotation syntax blessed by the Ruby core team, and Matz in
   particular (the lead of the Ruby project).
-- RBS supports certain enticing features, like method overloading and duck
```

Comment:
* Sorbet support overloading in RBI files.
* RBS is using nominal typing with its interfaces and you still have to include them like we do with Sorbet.